commands/init.go

```diff
@@ -151,3 +157,20 @@ func InitWebSocketClient() (*model.WebSocketClient, error) {
 	}
 	return client, nil
 }
+
+func InitUnixClient() (*model.Client4, error) {
+	httpClient := &http.Client{
+		Transport: &http.Transport{
+			Dial: func(network, addr string) (net.Conn, error) {
+				return net.Dial("unix", viper.GetString("local-socket-path"))
```

Comment:
Would it make sense to pass the socket path as a param ? That's what `InitClient` does too. And it may help to test it better.

Reply:
Sure, it would be more consistent 👍